lesson_07/objects/objects_app/src/main/java/com/codedifferently/lesson7/Jordan_Eldridge/Animal.java

```diff
@@ -0,0 +1,121 @@
+package com.codedifferently.lesson7.Jordan_Eldridge;
+
+public class Animal {
+
+  public enum AnimalType {
+    MAMMAL,
+    BIRD,
+    REPTILE,
+    FISH,
+    AMPHIBIAN,
+    INSECT
+  }
+
+  private String name;
+  private boolean canBreathe;
+  private int age;
```

Comment:
Prefer to use java.time.Instant to set a birth date and then calculate the age from that.